PasarGuardNodeBridge/grpclib.py

```
@@ -420,21 +432,22 @@ async def _sync_batch_users(self, users: list[service.User]) -> list[service.Use
         """Sync users via gRPC SyncUser stream. Returns failed users."""
         failed = []
         try:
-            async with self._client.SyncUser.open(metadata=self._metadata) as stream:
-                for user in users:
+            async with self._open_grpc_stream(self._client.SyncUser, self._internal_timeout) as stream:
+                for index, user in enumerate(users):
                     try:
                         await asyncio.wait_for(stream.send_message(user), timeout=self._internal_timeout)
                     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)